src/main/java/com/sourcery/gymapp/backend/workout/dto/ExerciseSimpleDto.java

```diff
@@ -0,0 +1,9 @@
+package com.sourcery.gymapp.backend.workout.dto;
+
+import java.util.UUID;
+
+public record ExerciseSimpleDto(
```

Comment:
I guess its still better to return all information of exercises, maybe we would like to have option to read more information about our exercise whenever we get lists of our exercises